test.js

```diff
@@ -2,17 +2,16 @@
  * @typedef {import('hast').Root} Root
  */
 
-import test from 'tape'
+import {test} from 'node:test'
+import {equal, ifError, fail} from 'node:assert/strict'
```

Comment:
```suggestion
import test from 'node:test'
import assert from 'node:assert/strict'
```

Hmm I prefer using the default exports for most Node things that export objects. Tree shaking isn’t a thing anyway. Particularly the assert one is important to me: it’s just easier to have one object with all the different functions on it, rather than the ton of functions taking up space in the scope?

Reply:
Sure, updated